stable/provazio/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.controller.enabled }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "provazio.controller.name" . }}
+  labels:
+    app: {{ template "provazio.controller.name" . }}
+    chart: {{ template "provazio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      provazio/service: {{ template "provazio.controller.name" . }}
+  template:
+    metadata:
+      labels:
+        provazio/service: {{ template "provazio.controller.name" . }}
```

Comment:
Labels should be as we do with other templates (missing stuff used by Helm)

Reply:
Done